_posts/slow-down.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+header_image: /img/posts/slow-down/header.jpg
+title: "Lifehacking 101: slow down"
+author: Mauro Servienti
+synopsis: "We believe we're surrounded by people who accomplish many things, and we feel we do nothing. We struggle to keep up, and we achieve little to nothing. Why is that, and can we do something about it?"
+tags:
+- Time management
+---
+
+At the beginning of summer 2021, I had the pleasure to work remotely for a week from the friend's place pictured in the header of this article.
+
+One of the prominent beauties of the place is the private, 50 meters-long swimming pool the apartment complex has. We swam every day before lunch.
+
+One day, in between lengths, I complained a bit because I was tired. That day's training was more challenging than expected. 
+
+My friend looked at me and asked: Why do you think we train? To train more tomorrow, and then more, and more, and so on indefinitely, he answered. There is no end goal. Just train more.
+
+To some extent, it was enlightening.
+
+Recently during one of my one-on-ones with Udi Dahan, the CEO of Particular Software, and yes, everyone has a quarterly one-on-one with Udi, we touched on something that later I connected to the discussion with my friend.
+
+## Slow is Smooth. Smooth is Fast
+
+With Udi, we discussed the "Slow is Smooth. Smooth is Fast" SEAL and Delta Force motto. The motto can be summarized as _you cannot go fast without first slowing down_. I quickly connected it to my swimming activities.
+
+Swimming is an extraordinarily complex and challenging activity. Swimming fast comes with many subtle challenges. Technique is everything; it's much more important than strength. And probably the most difficult part is breathing.
+
+When swimming front crawl or butterfly for the sake of the sample, the breath pace is determined by strokes pace, and swimmers cannot breath when their body needs or at the rhythm they want. Usually, what happens to amateur swimmers when they start is that they try to swim faster by stroking faster. Stroking faster comes with breathing at a faster pace and with less time for each breath. Less time to breathe means less oxygen exchange which causes hypoxia that eventually leads to less effective muscles. The result is that they go slower instead of faster.
```

Comment:
```suggestion
When swimming front crawl or butterfly for the sake of the sample, the breath pace is determined by the stroke pace, and swimmers cannot breath when their body needs it or at the rhythm they want. Usually, what happens to amateur swimmers when they start is that they try to swim faster by stroking faster. Stroking faster comes with breathing at a faster pace and with less time for each breath. Less time to breathe means less oxygen exchange which causes hypoxia that eventually leads to less effective muscles. The result is that they go slower instead of faster.
```